src/client/components/app/Navbar/index.tsx

```diff
@@ -109,9 +117,22 @@ export const Navbar = ({
     Icon: getNetworkIcon(network),
   }));
 
+  const secondTitleEnabled = !!subTitle?.length;
+
+  const vaultText = (
+    <>
+      &nbsp;/&nbsp;<StyledText>{subTitle}</StyledText>
```

Comment:
Is `&nbsp;/&nbsp;` needed? Perhaps better to create a component with margin around /

Reply:
I would leave nbsp since it is the real letter spacing related to that type of font and the current font-size. the margins will break the correct spacing if you change the font or size.